tests/unit/types/arrays/test_documentarray.py

```diff
@@ -397,3 +397,17 @@ def test_cache_invalidation_sort_reverse(docarray_for_cache):
     docarray_for_cache.reverse()
     assert docarray_for_cache[0].id == '2'
     assert docarray_for_cache[1].id == '1'
+
+
+def test_sample():
```

Comment:
I think you can break this unit test into two: one testing that correct amount of items are returned, another testing that setting random seed works as expected

Reply:
true, PR updated